patches/4D530910 - Forza Motorsport 4.patch

```diff
@@ -1,13 +1,12 @@
 title_name = "Forza Motorsport 4"
 title_id = "4D530910"
 hash = "7566081090ECD4EC"
-#media id 26FF1101
+#media_id = "26FF1101"
```

Comment:
Commented media id seem kinda useless now that we have executable hash

Reply:
Should they all be removed?

Reply:
Yes.